runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -2975,6 +3048,17 @@ void chpl_comm_impl_regMemPostAlloc(void* p, size_t size)
   mem_region_t* mr;
   int mr_i;
 
+#if 0
```

Comment:
I don't love the `if 0` uses, I would prefer we just use a new define to trigger this. i.e. `#if PERFSTATS_COMM_UGNI_ZERO_ARRAY_ALLOC` or something